docs/sources/release-notes.md

```diff
@@ -89,4 +45,4 @@ impact on users. This list will be updated as issues are resolved.
 An idiosyncrasy in AUFS prevents permissions from propagating predictably
 between upper and lower layers. This can cause issues with accessing private
 keys, database instances, etc. For complete information and workarounds see
-[Github Issue 783](https://github.com/docker/docker/issues/783).
\ No newline at end of file
+[Github Issue 783](https://github.com/docker/docker/issues/783).
```

Comment:
Is that the styling issue? I think I tried some CSS changes a while ago that fixed it, but need to look if I made notes. Or are there other issues?